doc/devel/documenting_mpl.rst

```diff
@@ -88,16 +93,89 @@ errors.  To unset it, use
 
 You can use the ``O`` variable to set additional options:
 
-* ``make O=-j4 html`` runs a parallel build with 4 processes.
-* ``make O=-Dplot_formats=png:100 html`` saves figures in low resolution.
-* ``make O=-Dplot_gallery=0 html`` skips the gallery build.
+.. code-block:: sh
+    
+    #runs a parallel build with 4 processes.
```

Comment:
I'd go for ``# run a parallel build with 4 processes``, i.e. use an imperative style rather than a statement, similar to how we write docstrings. The same holds for the following two comments.

Please also add a space between `#` and text (multiple places throughout the PR).